G/Glib/build_tarballs.jl

```diff
@@ -2,6 +2,8 @@ using BinaryBuilder
 
 name = "Glib"
 version = v"2.84.0"
+# We bumped the Yggdrasil version number to add a compat bound for PCRE2
+ygg_version = v"2.84.1"
```

Comment:
Why not using the real [v2.84.1](https://gitlab.gnome.org/GNOME/glib/-/releases/2.84.1)?